src/Cmfd.cpp

```diff
@@ -446,11 +480,16 @@ void Cmfd::computeDs(int moc_iteration) {
             d_hat = 2.0 * d * f * d_next * f_next / (length_perpen
                     * d * f + next_length_perpen * d_next*f_next);
```

Comment:
be consistent with the spacing here (per https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Horizontal_Whitespace separating operators with spaces is the default but can be removed for terms i.e. ``next_length_perpen*d_next*f_next`` but I don't think ``next_length_perpen * d_next*f_next`` is correct style.
t I don't think ``next_length_perpen * d_next*f_next`` is correct styl